WordPress/Classes/ViewRelated/Blog/Blog List/BlogListViewController+SiteCreation.swift

```diff
@@ -1,20 +0,0 @@
-extension BlogListViewController {
-    @objc
-    func launchSiteCreation() {
```

Comment:
This function is used as an observer to handle "create site" notification. I'm not sure if removing this observer would break some workflow.

https://github.com/wordpress-mobile/WordPress-iOS/blob/7b0fb0b1e2391850c1248dad4421412a888ae893/WordPress/Classes/ViewRelated/Blog/Blog%20List/BlogListViewController.m#L563-L568

https://github.com/wordpress-mobile/WordPress-iOS/blob/7b0fb0b1e2391850c1248dad4421412a888ae893/WordPress/Classes/ViewRelated/NUX/Post%20Signup%20Interstitial/PostSignUpInterstitialViewController.swift#L10

https://github.com/wordpress-mobile/WordPress-iOS/blob/20ad8b877c47a2f2eb74c96f376a822b95e403a4/WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift#L634

Reply:
Good call. I haven't tested it, but yes, it will obviously break this part of the flow.
 
This notification is only used from the following screen that's shown after account creation:

<img width="300" alt="Screenshot 2024-07-30 at 10 35 38 AM" src="https://github.com/user-attachments/assets/17d3ef07-560b-4707-97fd-97cf45d8182b">

It duplicates the functionality of the new-ish placeholder on the home page:

<img width="300" alt="Screenshot 2024-07-30 at 10 38 17 AM" src="https://github.com/user-attachments/assets/ca4db790-ddb5-4692-b376-6f754cec9aac">

Since this code is largely unmaintainable and duplicates the existing functionality, I think it's an easy call to remove it so we don't have to worry about it anymore. @jkmassel, you agree?

Reply:
(got it OK-ed in DMs)